tests/integration/grpc/ocm_invitation_test.go

```diff
@@ -106,23 +107,43 @@ var _ = Describe("ocm invitation workflow", func() {
 		einstein        = &userpb.User{
 			Id: &userpb.UserId{
 				OpaqueId: "4c510ada-c86b-4815-8820-42cdf82c3d51",
-				Idp:      "cernbox.cern.ch",
+				Idp:      "https://cernbox.cern.ch",
 				Type:     userpb.UserType_USER_TYPE_PRIMARY,
 			},
 			Username:    "einstein",
 			Mail:        "einstein@cern.ch",
 			DisplayName: "Albert Einstein",
 		}
+		federatedEinstein = &userpb.User{
+			Id: &userpb.UserId{
+				Type:     userpb.UserType_USER_TYPE_FEDERATED,
+				Idp:      "cernbox.cern.ch",
```

Comment:
can you add a little comment why the federated user has no protocol contained int the Idp field.

Reply:
hm, I think that should be part of the documentation ...